src/ui/layout/message-layout.svelte

```diff
@@ -85,7 +59,7 @@
 
   @container layout-query-container (width >= 768px) {
     .message-layout {
-      min-height: 440px;
+      min-height: 354px;
```

Comment:
Agreed with Iga, this aligns the button a lot better in most cases. We'll work in a separate ticket to properly align the button with the base of the left panel. But that will take more refactoring and we want to get this shipped first.